### PR TITLE
feat(validation-questionnaire-three): Create validation questionnaire level three

### PR DESCRIPTION
**Pull request by doohanas**
## Changes

1. Create validation questionnaire level three
2. Refactor the layout of the `questionnaire-three` component

## Preview

1. Create validation questionnaire level three
validation-quest-3

## Evidence
title: feat(validation-questionnaire-three): Create validation questionnaire level three
project: Desa Digital
participants: @doohanas @yoslie @agunghide @maulanayuseph @naufalihsank

### CODE REVIEW

File: components/Questionnaire/index.vue [hunk 1]
```diff
@@ -121,9 +121,11 @@ export default {
       isLevelOne: true,
       isLevelTwo: false,
       isLevelThree: false,
+      isLevelFour: false,
       showLevelOne: true,
       showLevelTwo: false,
       showLevelThree: false,
+      showLevelFour: false,
```
Comment (yoslie) [inline components/Questionnaire/index.vue:128]: unused code?

File: components/Questionnaire/index.vue [hunk 2]
```diff
@@ -157,12 +162,30 @@ export default {
         this.onSubmit()
       }
     },
+    onNextLevelThree (properties) {
+      this.params.properties.tentang_bumdes = properties.tentang_bumdes
+      this.params.properties.potensi_desa = properties.tentang_bumdes
```
Comment (yoslie) [inline components/Questionnaire/index.vue:167]: why assign the same value? maybe another one should be `potensi_desa` ?

**Review by yoslie — APPROVED** [review]

File: components/Questionnaire/index.vue (hunk 2 above)
Comment (doohanas) [reply components/Questionnaire/index.vue]: I think it's wrong kak, you can see the changes [here](https://github.com/jabardigitalservice/dedi-landing-page/pull/94/commits/e0d3665516e6f4466539a34050520058f19dcfab)

File: components/Questionnaire/index.vue (hunk 1 above)
Comment (doohanas) [reply components/Questionnaire/index.vue]: Yes, it's not used and the changes [here](https://github.com/jabardigitalservice/dedi-landing-page/pull/94/commits/ae51bc0772cac27f7c9499339331bbb522f935da)

**Review by naufalihsank — APPROVED** [review]

**Review by agunghide — APPROVED** [review]

**Review by maulanayuseph — APPROVED** [review]

**Pull request merged**